lib/Command/Updater/AfterUpdate.php

```diff
@@ -42,14 +43,22 @@ protected function configure()
     {
         $this->setName('news:updater:after-update')
             ->setDescription('removes old read articles which are not starred')
-            ->addArgument('purge_count', InputArgument::OPTIONAL, 'The amount of items to purge');
+            ->addArgument('purge-count', InputArgument::OPTIONAL, 'The amount of items to purge')
+            ->addOption('purge-unread', null, InputOption::VALUE_NONE, 'If unread items should be purged');
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
-        $count = (int) $input->getArgument('purge_count');
+        $count        = (int) $input->getArgument('purge-count');
```

Comment:
`purge-count` is optional so casting it to int will result in a 0, which prevents the app-config fallback to `autoPurgeCount` to trigger.